### ISSUE TITLE
no predict mask 

### ISSUE THREAD

**Issue by zzllc3** [matterport/Mask_RCNN#396]
@waleedka.When I use "mask_rcnn_balloon.h5" to splash, everything goes perect,but when I use the model I trained which based on coco,I can get the predict-bboxes and scores but no masks!
I changed nothing except model path,Do you know the reason,Thanks a lot

**Comment by filipduch** [comment]
Same here. I trained my own model and balloon and none of them is returning masks.

**Comment by juruoWYC** [comment]
Same here!!! I want to know the reason.

**Comment by dennissv** [comment]
I'm also experiencing this problem.

**Comment by caozi** [comment]
I have the same problem!

**Comment by dennissv** [comment]
Just to clarify a bit on this problem, masks are returned it's just that they're all blank. Masks visualize correctly when loading the dataset so the problem is only when running inference. Also the bbox generation looks really good when doing inference. During training the loss for mrcnn_mask drops very low (<-1e6).

**Comment by waleedka** [comment]
Hmm. That's odd. If I understand correctly, using a trained model works, but training a new one doesn't. Is that right? 

I would need more details to track it:
1. Can you try the train_shapes notebook? Let me know if this one shows any masks. 
2. Which version are you using? The 2.1 release or the latest master branch?

**Comment by dennissv** [comment]
Using the latest mater branch the balloon example seems to work but training a new model does not.

Testing some earlier versions I think it breaks at 38ed339 (or possibly dffcb61 can't tell because I can't train it more than ~3 epochs). The latest working version that I can train with and everything seems to work with is c7e3aa7.

**Comment by waleedka** [comment]
Can you confirm if the train_shapes notebook trains correctly for you on the master branch?

**Comment by dennissv** [comment]
Yes, train_shapes notebooks does train correctly on master branch.

**Comment by juruoWYC** [comment]
It works on the 2.1 release, but not on the latest master branch.

**Comment by chengchu88** [comment]
hello, waleedka,
I downloaded the zip on Apr 27 and still have the same issue. I re-trained and modified the balloon with my own dataset, got scores and bbox but mask is all 'False'.
Can  you point some direction for debugging?
thank you

**Comment by hoangphucITJP** [comment]
> Just to clarify a bit on this problem, masks are returned it's just that they're all blank. Masks visualize correctly when loading the dataset so the problem is only when running inference. Also the bbox generation looks really good when doing inference. During training the loss for mrcnn_mask drops very low (<1e-6).

@dennissv, which backbone did you use to archive that low loss?
